packages/flutter/lib/src/widgets/app.dart

```diff
@@ -1187,8 +1188,22 @@ class WidgetsApp extends StatefulWidget {
   /// and view what widgets and render objects associated with it. An outline of
   /// the selected widget and some summary information is shown on device and
   /// more detailed information is shown in the IDE or DevTools.
+  @Deprecated(
+    'Use debugShowWidgetInspectorOverrideNotifier.value instead. '
+    'This feature was deprecated after v3.13.0-19.0.pre.'
+  )
   static bool debugShowWidgetInspectorOverride = false;
```

Comment:
change this to be a getter + setter pair that reads and writes to
debugShowWidgetInspectorOverrideNotifier.